packages/marketing/scripts/vercel.sh

```
@@ -1,3 +1,4 @@
+(cd ../client && pnpm run build)
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)